src/generate.js

```diff
@@ -17,11 +17,15 @@ self.generate = function(options) {
 	if (!options.hasOwnProperty('uppercase')) options.uppercase = true;
 	if (!options.hasOwnProperty('excludeSimilarCharacters')) options.excludeSimilarCharacters = false;
 
+	// check if must limit the number of repeating characters
+	if (!options.hasOwnProperty('repeatCharactersLimit') || typeof options.repeatCharactersLimit !== 'number' || options.repeatCharactersLimit < 2) options.repeatCharactersLimit = undefined;
```

Comment:
You don't need to sanitize the passed option here (checking for number and value). Also, a repeat character limit of 0 seems to make the most sense, rather than undefined for no limit.